backend/examples/user_custom_properties/default/locale/pt.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Tagline": "Slogan",
+  "Localization": "Localização",
+  "Hobbies": "Passatempos",
+  "Personal Website": "Website pessoal",
+  "Birthday": "Data de nascimento",
+  "Telephone": "Telefone",
```

Comment:
Is it `Telephone` or `Phone` or `Phone number` ?

Reply:
```suggestion
  "Phone": "Telefone",
```